components/challenge_bypass_ristretto_ffi/BUILD.gn

```diff
@@ -0,0 +1,14 @@
+import("//brave/build/cargo.gni")
+
+cargo_build("challenge_bypass_ristretto") {
+  manifest_dir = "//brave/vendor/challenge-bypass-ristretto-ffi"
+  inputs = [
+    "build.rs",
+    "Cargo.toml",
+    "src/lib.h",
+    "src/lib.rs",
+    "src/wrapper.cpp",
+    "src/wrapper.hpp"
+  ]
+  output = "$root_gen_dir/challenge_bypass_ristretto/out"
```

Comment:
this should be the actual lib so `$root_gen_dir/challenge_bypass_ristretto/out/challenge_bypass_ristretto.a` (you'll need platform specific outputs). In the python script you can get the directory from the full lib path, but you have to specify the actual output lib here or you won't be able to link it in other targets because GN will complain that no target provides it

Reply:
After discussing we had to have both lib_path and build_path, changes have been made